subsys/bluetooth/mesh/provisioner.c

```diff
@@ -724,8 +717,8 @@ int bt_mesh_prov_remote_pub_key_set(const uint8_t public_key[64])
 	}
 
 	/* Swap X and Y halves independently to big-endian */
-	memcpy(&bt_mesh_prov_link.conf_inputs[81], public_key, 32);
-	memcpy(&bt_mesh_prov_link.conf_inputs[81 + 32], &public_key[32], 32);
+	sys_memcpy_swap(&bt_mesh_prov_link.conf_inputs[81], public_key, 32);
+	sys_memcpy_swap(&bt_mesh_prov_link.conf_inputs[81 + 32], &public_key[32], 32);
```

Comment:
Can you put this in a separate commit?

Reply:
https://github.com/zephyrproject-rtos/zephyr/blob/efccfcbfc91501cac2ede0541a48c338f2b0cae4/subsys/bluetooth/mesh/provisioner.c#L414-L415

Have done the same thing here?